webpack.config.js

```diff
@@ -92,12 +98,58 @@ const createCssLoaderConfig = () =>
 
 const createCssExtractTextPlugin = () => new ExtractTextPlugin(CSS_FILENAME_OUTPUT_PATTERN);
 
+class PostCompilePlugin {
+  constructor(fn) {
+    this.fn = fn;
+  }
+
+  apply(compiler) {
+    compiler.plugin('done', (stats) => this.fn(stats));
+  }
+}
+
+const createStaticBuildPlugin = () => {
+  return new PostCompilePlugin(() => {
+    if (!BUILD_STATIC_DEMO_ASSETS || !fs.existsSync(OUT_DIR_ABS)) {
+      return;
+    }
+
+    const demosDirAbs = path.resolve('./demos');
+    const tmpDirAbs = fs.mkdtempSync(path.join(os.tmpdir(), 'mdc-web-demo-output-'));
```

Comment:
Is there a reason this ends with a dash?

Reply:
Yes - [`mkdtempSync`](https://nodejs.org/api/fs.html#fs_fs_mkdtemp_prefix_options_callback) appends 6 random characters to the end of the `prefix` string passed in.